stdlib/transactions/src/main/ballerina/transactions/2pc_transaction.bal

```diff
@@ -62,7 +63,8 @@ type TwoPhaseCommitTransaction object {
                         ret = {message:OUTCOME_HAZARD};
                     }
                     NotifyResult => {
-                        boolean localCommitSuccessful = commitResourceManagers(self.transactionId, self.transactionBlockId);
+                        boolean localCommitSuccessful = commitResourceManagers(self.transactionId, self.
```

Comment:
I would rather break it like this:
boolean localCommitSuccessful = 
                                     commitResourceManagers(self.transactionId, self.transactionBlockId);